pyQuARC/schemas/rule_mapping.json

```
@@ -4591,5 +4797,75 @@
         },
         "severity": "info",
         "check_id": "date_compare"
+    },
+    "granule_single_date_time_check": {
+        "rule_name": "Granule Single Date Time Check",
+        "fields_to_apply": {
+            "echo-g": [
+                {
+                    "fields": [
+                        "Granule/Temporal/RangeDateTime/BeginningDateTime",
+                        "Granule/Temporal/RangeDateTime/EndingDateTime"
+                    ],
+                    "relation":"neq"
+                }
+            ]
+        },
+        "severity": "warning",
+        "check_id": "date_compare"
+    },
+    "granule_project_short_name_check": {
+        "rule_name": "Granule Project Short Name Check",
+        "fields_to_apply": {
+            "echo-g": [
+                {
+                    "fields": [
+                        "Granule/Campaigns/Campaign/ShortName",
+                        "Granule/Collection/DataSetId",
+                        "Granule/Collection/ShortName",
+                        "Granule/Collection/VersionId"
+                    ]
+                }
+            ],
+            "umm-g": [
+                {
+                    "fields": [
+                        "Projects/ShortName",
+                        "CollectionReference/EntryTitle",
+                        "CollectionReference/ShortName",
+                        "CollectionReference/Version"
+                    ]
+                }
+            ]
+        },
+        "severity": "info",
+        "check_id": "granule_project_short_name_check"    
+    },
+    "granule_sensor_short_name_check": {
+        "rule_name": "Granule Sensor Short Name Check",
+        "fields_to_apply": {
+            "echo-g": [
+                {
+                    "fields": [
+                        "Granule/Platforms/Platform/Instruments/Instrument/Sensors/Sensor/ShortName",
+                        "Granule/Collection/DataSetId",
+                        "Granule/Collection/ShortName",
+                        "Granule/Collection/VersionId"
+                    ]
+                }
+            ],
+            "umm-g": [
+                {
+                    "fields": [
+                        "Platforms/Instruments/ComposedOf/ShortName",
+                        "CollectionReference/EntryTitle",
+                        "CollectionReference/ShortName",
+                        "CollectionReference/Version"
+                    ]
+                }
+            ]
+        },
+        "severity": "info",
+        "check_id": "granule_sensor_short_name_check"    
     }
-}
+}
```

```suggestion
}

```
Suggestion to add a new line to the end of the file